packages/mds-transactions-api/handlers/get-transactions.ts

```diff
@@ -7,19 +7,31 @@ import { TransactionApiRequest, TransactionApiResponse } from '../@types'
 export type TransactionApiGetTransactionsRequest = TransactionApiRequest &
   ApiRequestParams<'provider_id' | 'start_timestamp' | 'end_timestamp'>
 
-export type TransactionApiGetTransactionsResponse = TransactionApiResponse<{ transactions: TransactionDomainModel[] }>
+export type TransactionApiGetTransactionsResponse = TransactionApiResponse<{
+  transactions: TransactionDomainModel[]
+  links: PaginationLinks
+}>
 
 export const GetTransactionsHandler = async (
   req: TransactionApiGetTransactionsRequest,
   res: TransactionApiGetTransactionsResponse
 ) => {
   try {
-    const transactions = await TransactionServiceClient.getTransactions({
+    const { transactions, cursor } = await TransactionServiceClient.getTransactions({
       ...parseRequest(req).single({ parser: String }).query('provider_id'),
       ...parseRequest(req).single({ parser: Number }).query('start_timestamp', 'end_timestamp')
     })
     const { version } = res.locals
-    return res.status(200).send({ version, transactions })
+    // convert Cursor to links
+    // TODO finish implementing
+    const links: PaginationLinks = {
+      first: 'first?',
+      last: 'last?',
+      prev: cursor.beforeCursor,
+      next: cursor.afterCursor
+    }
+    console.log('!!!!', JSON.stringify(links))
```

Comment:
assuming this is just for debugging, reminder to remove this and other logs that aren't needed before merging